apps/frontend/src/libs/components/select/libs/types/option.type.ts

```diff
@@ -0,0 +1,6 @@
+type Option = {
```

Comment:
```suggestion
type SelectOption = {
```
should be in `src/libs/types`